packages/nimble-components/CONTRIBUTING.md

```diff
@@ -387,6 +387,16 @@ Test utilties located in [`/src/utilities/tests`](/packages/nimble-components/sr
 
 The jasmine unit tests utilize [`fixture.ts`](/packages/nimble-components/src/utilities/tests/fixture.ts) for component tests. The fixture utility gives tools for managing the component lifecycle. For some usage examples see [`fixture.spec.ts`](/packages/nimble-components/src/utilities/tests/fixture.spec.ts).
 
+### Disabling tests
+
+If a test is failing on a specific browser but passing on others, it is possible to temporarily mark it to be skipped for that browser by applying the tag `#failsFirefox`, `#failsWebkit`, or `#failsChrome` to the test name:
+
+```ts
+it('#failsFirefox sets title when cell text is ellipsized', ...);
```

Comment:
Update the recommendation to move the tag to the end of the name

Reply:
Done